CONTRIBUTING.md

```diff
@@ -23,7 +23,7 @@ Get it all applied via:
 
 ```shell
 poetry run isort .
-poetry run autopep8 --in-place -r .
+poetry run flake8 cyclonedx/ tests/
```

Comment:
autopep8 fixed code..this was removed because of dependency conflicts, i see.
having flake8 instead, to at least display the issues, so they can be fixed manually --- :+1: